src/App.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import Router from './router';
-import { Footer } from './components';
+import { Footer, NavBar } from './components';
 
 function App() {
   return (
     <div className="App" style={styles.all}>
+      <NavBar />
```

Comment:
The reason the navbar wasnt included in the file is because I wanted the homepage including the navbar be a perfect 100vh for any screen size. In order to do that homepage needs to render the navbar alongside the main header.

Ideally we would have it like this because it makes more sense to not re-render the navbar everytime someone switches a page, but idk how to do that

Reply:
We should handle the re-rendering issue in another ticket. However, when I try different window sizes it appears to always load the first page to perfectly fit the vh. You never see the white background of the section you scroll down into when you load the homepage

Reply:
Yes, that is because the height of the red section is 100vh + the height of the navbar, if you scroll down on your branch's homepage there is additional red space, whereas on main, as soon as you start scrolling down it becomes white

Reply:
I thought about this some more, and I want to find a better solution for this issue. So lets keep what you have, and I'll fix it up later, because currently it also has issue with centering the components which is bugging me.